CDS Maps/Individual Maps/CDS-to-FHIR/nNGM-CDS-to-FHIR-Systemische Therapie.map

```diff
@@ -538,7 +497,7 @@ group CreatePatientSY(source operations: BackboneElement, target bundle: Bundle,
 group TransformPatientSY(source operations: BackboneElement, target tgt: Patient, target composition: Composition, target section: BackboneElement)
 {
     // Profile url
-    operations -> tgt.meta as meta collate, meta.profile = 'http://uk-koeln.de/fhir/StructureDefinition/Patient/nNGM';
+    operations -> tgt.meta as meta collate, meta.profile = 'http://uk-koeln.de/fhir/StructureDefinition/Patient/nNGM/patient';
```

Comment:
The transformation group for the patient can be deleted as well

Reply:
410